_sources/governance.rst.txt

```diff
@@ -0,0 +1,169 @@
+PySAL project governance
+========================
+
+The purpose of this document is to formalize the governance process used by the PySAL project in both ordinary and extraordinary situations, and to clarify how decisions are made and how the various elements of our community interact, including the relationship between open source collaborative development and work that may be funded by for-profit or non-profit entities.
+
+
+The Project¶
+------------
+
+PySAL, the Python spatial analysis library, is an open source cross-platform project (The Project). The goal of The Project is to develop open source software for geospatial data science in Python, and in particular the PySAL library and its Sub-Packages. The Software developed by The Project is released under the BSD (or similar) open source license, developed openly and hosted on public GitHub repositories under the pysal GitHub organization.
```

Comment:
... PySAL library and its component modules (hereinafter referred to as "Sub-Packages").

Reply:
s/Sub-Packages/Packages/